goon/browserassets/css/browserOutput.css

```
@@ -437,3 +444,33 @@ span.body .coderesponses {color: #FF0000;}
 .announcement.sec h1 { color: #f00; font-size: 180%; font-family: Verdana, sans-serif; }
 
 .bolditalics {font-style: italic; font-weight: bold;}
+
+.boxed_message {
+	text-align: center;
+	border-style: solid;
+	border-width: medium;
+	border-color: black;
+	border-radius: 5px;
+	margin: 25px;
+	padding: 15px;
+  }
```

```suggestion
}
```
Same thing above for the comment relating to rules in _browserOutput-dark.css_ Can you go through and fix the formatting on the new rules that you created?